src/main/java/org/folio/circulation/domain/CheckInContext.java

```diff
@@ -240,4 +267,8 @@ public ZonedDateTime getCheckInProcessedDateTime() {
   public ItemStatus getItemStatusBeforeCheckIn() {
     return itemStatusBeforeCheckIn;
   }
+
+  public ZoneId getTimeZone() {
+    return timeZone;
+  }
```

Comment:
Same here, can use `@Getter` annotation instead and remove other trivial getters. Again, you can check `LoanAndRelatedRecords` as one of the examples.

Reply:
Done